video_xblock/tests/unit/test_mixins.py

```diff
@@ -450,6 +450,17 @@ def test_route_transcripts(self):
                 self.xblock, 'srt_to_vtt', query='test-trans.srt'
             )
 
+    def test_vtt_to_text(self):
```

Comment:
you should also add a case where the conversion is skipped if vtt_content is not in bytes if you use that isinstance check